tools/settings_template_readonly.py

```diff
@@ -45,6 +45,13 @@
 #
 # JS_ENGINES = [NODE_JS] # add V8_ENGINE or SPIDERMONKEY_ENGINE if you have them installed too.
 #
+# WASMER = os.path.expanduser(os.path.join('~', '.wasmer', 'bin', 'wasmer'))
+# WASMTIME = os.path.expanduser(os.path.join('~', 'wasmtime'))
```

Comment:
Do we need these two variables?  Can we just add WASM_ENGINES and be done with it?  It looks like we are detecting that type of the engine based on the filename anyway in jsrun.py

Reply:
This basically just parallels the above code for `SPIDERMONKEY_ENGINE` and `V8_ENGINE`.

I think it makes sense there, and here, because we do need to know the identities of these engines. For example, v8 requires `--` before the program args, and wasmer requires `run` before the wasm name.